### DISCUSSION TITLE
프로덕션 데이버테이스의 컬럼 타입 수정을 어떻게 진행해야 할까요?

### DISCUSSION THREAD

**Discussion by 49EHyeon42** [100-hours-a-week/discussions#64] [개발] [answered]
저희 커뮤니티 기능 중 내용 작성에서 이슈가 발생했습니다. mariadb 기준 TEXT가 아닌 VARCHAR(255)로 타입이 지정되어 명세와 달리 긴 문자열을 받을 수 없는 상황입니다.

핫픽스 브랜치를 만들어서 JPA를 통해 TEXT 타입을 지정할 수 있도록 수정하고, DataGrip으로 프로덕션 데이터베이스에 접근하여 문제가 되는 타입을 수정해도 괜찮을까요??

**Comment by ej31** [comment, marked as answer]
# 답변
- 디비에서 형변환을 진행할 때 작은사이즈에서 큰 사이즈로 넘어갈 때는 별 문제가 없습니다.
- 큰 사이즈에서 작은 사이즈로 내릴 때에는 문제가 될 수 있지요
- 자바의 캐스팅을 생각해보시면 간단합니다

- 결론은 VARCHAR 보다 TEXT 가 더 크기 때문에 문제 없습니다!
